### PR TITLE
[16082] Adjust height between messages to match designs

### LINKED ISSUE #16082: Messaging - spacing between single messages is too much
Currently the spacing for single messages is too big, see the photo

Composer_for_Mobile_Figma_2023-05-30_13-25-16

Review the designs and ensure they are correct:
https://www.figma.com/file/wA8Epdki2OWa8Vr067PCNQ/Composer-for-Mobile?type=design&node-id=5481-354129&t=zc32OqUZuDYsU1VJ-4

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #16082 
status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/messages/content/view.cljs [hunk 1]
```diff
@@ -114,7 +114,7 @@
                                      (js/setTimeout #(reset! show-delivery-state? false)
                                                     delivery-state-showing-time-ms))))
           :on-long-press       #(on-long-press message-data context)}
-         [rn/view {:style {:padding-vertical 8}}
+         [rn/view {:style {:padding-vertical 4}}
```
Comment (J-Son89) [inline src/status_im2/contexts/chat/messages/content/view.cljs:115]: Can you please verify these visual changes using iPhone 11 Pro (or iPhone 12 Pro) and Figma? Let me know if you need any help with this 👍

Comment (ibrkhalil) [reply src/status_im2/contexts/chat/messages/content/view.cljs:115]: I only have 13 Pro currently, I tested it on pixel 7A.
Will post a screen shot for how it looks on 13 Pro

Comment (ibrkhalil) [reply src/status_im2/contexts/chat/messages/content/view.cljs:115]: image

**Review by ilmotta — APPROVED** [review]
Thanks for yet another fix @ibrkhalil. Code seems good, but I haven't tested how it looks.

**Comment by VolodLytvynenko** [comment]
@ibrkhalil Could you please rebase the PR and resolve existing conflicts. After that we can perform manual testing. Thanx!

**Comment by ibrkhalil** [comment]
> @ibrkhalil Could you please rebase the PR and resolve existing conflicts. After that we can perform manual testing. Thanx!

Done

**Comment by VolodLytvynenko** [comment]
hi @ibrkhalil thank you for your work. No issues from my side. PR can be merged

**Pull request merged**